src/app/app-routing.module.ts

```diff
@@ -1,12 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { LoginGuard } from './core/Guards/login.guard';
+import { UnloginGuard } from './core/Guards/unlogin-guard.service';
 
 
 const routes: Routes = [
-  {path: 'home', loadChildren: () => import('./home/home.module').then(m => m.HomeModule)},
+  {path: 'home', loadChildren: () => import('./home/home.module').then(m => m.HomeModule), canActivate: [UnloginGuard]},
```

Comment:
canActivate is used to prevent unauthorized users from accessing certain routes. See docs for more info.

canLoad is used to prevent the application from loading entire modules lazily if the user is not authorized to do so.